src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -34,12 +34,6 @@
  */
 public final class TokenTypes {
 
-    // The following three types are never part of an AST,
-    // left here as a reminder so nobody will read them accidentally
-
-    // These are the types that can actually occur in an AST
-    // it makes sense to register Checks for these types
-
     /**
```

Comment:
These comments are out of date. The code they are referring to was deleted some time ago.